utils.h

```diff
@@ -143,6 +143,13 @@ gboolean janus_flags_is_set(janus_flags *flags, gsize flag);
  * @note A failure may indicate that creating any of the subdirectories failed: some may still have been created */
 int janus_mkdir(const char *dir, mode_t mode);
 
+/*! \brief Helper to convert \a path relative to \a base_dir to absolute path.
+ *  If \a path already represents absolute path then just g_strdup it.
+ * @param[in] base_dir Path wich will be prepended to \a path if it's relative
```

Comment:
Not sure what `\a` is in Doxygen, but we don't use it anywhere: if it's meant to look like code, we use `\c` instead.